lib/rules/i18n-ellipsis.js

```diff
@@ -26,12 +38,37 @@ var rule = module.exports = function( context ) {
 
 			node.arguments.forEach( function( arg ) {
 				var argumentString = getTextContentFromNode( arg );
-				if ( argumentString && -1 !== argumentString.indexOf( '...' ) ) {
+				if ( argumentString && containsThreeDots( argumentString ) ) {
 					context.report( {
 						node: arg,
 						message: rule.ERROR_MESSAGE,
 						fix: function( fixer ) {
-							return fixer.replaceText( arg, argumentString.replace( /\.\.\./g, '…' ) );
+							if ( arg.type === 'TemplateLiteral' ) {
+								var fixes = [];
+								arg.quasis.forEach(
```

Comment:
Minor: Could accumulate this with `Array#reduce`

```js
return arg.quasis.reduce( ( fixes, quasi ) => {
	if ( /* ... */ ) {
		fixes.push( /* ... */ );
	}

	return result;
}, [] );

Reply:
❤️  nice, `reduce` in 0638c7cb7488624725d0348e3795cfe063704df7